tests/test_immutableset.py

```diff
@@ -269,3 +269,10 @@ def test_pickling(self):
         self.assertEqual(immutableset([5, 2]).__reduce__(), (immutableset, ((5, 2),)))
         self.assertEqual(immutableset([5]).__reduce__(), (immutableset, ((5,),)))
         self.assertEqual(immutableset().__reduce__(), (immutableset, ((),)))
+
+    def test_subtract_from_normal_set(self):
+        self.assertEqual({5, 6}, frozenset([4, 5, 6]) - immutableset([2, 3, 4]))
+        self.assertEqual({5, 6}, {4, 5, 6} - immutableset([2, 3, 4]))
+        self.assertEqual(set(), set() - immutableset())
+        self.assertEqual({2, 3}, {2, 3} - immutableset([]))
```

Comment:
Just to be consistent with the above:
```suggestion
        self.assertEqual({2, 3}, {2, 3} - immutableset())
```